action.yml

```diff
@@ -23,6 +27,7 @@ runs:
     - run: >
         wget
         '${{ inputs.url }}'
+        ${{ inputs.target && ('-O \'${{ inputs.target }}\'') }}
```

Comment:
There is a problem with the string interpolation, did a quick test run and saw this feature

```
Error: Swiddis/download-file/main/action.yml (Line: 27, Col: 12): Unexpected symbol: '${{'. Located at position 25 within expression: inputs.target && ('-O \'${{ inputs.target
Error: GitHub.DistributedTask.ObjectTemplating.TemplateValidationException: The template is not valid. Swiddis/download-file/main/action.yml (Line: 27, Col: 12): Unexpected symbol: '${{'. Located at position [25](https://github.com/peternied/security/actions/runs/6240719223/job/16941404988#step:8:25) within expression: inputs.target && ('-O \'${{ inputs.target
   at GitHub.DistributedTask.ObjectTemplating.TemplateValidationErrors.Check()
   at GitHub.Runner.Worker.ActionManifestManager.ConvertRuns(IExecutionContext executionContext, TemplateContext templateContext, TemplateToken inputsToken, String fileRelativePath, MappingToken outputs)
   at GitHub.Runner.Worker.ActionManifestManager.Load(IExecutionContext executionContext, String manifestFile)
Error: Fail to load Swiddis/download-file/main/action.yml
```

- Commit to use this version https://github.com/opensearch-project/security/compare/main...peternied:security:test-download-file-update
- Results https://github.com/peternied/security/actions/runs/6240719223/job/16941404988

Reply:
My bad for not testing it -- Can't find a good reference for doing conditional flags so I just split it into two separate `run` commands. Test run for new version here: https://github.com/Swiddis/dashboards-observability/actions/runs/6241980661/job/16945468606